app/javascript/packages/document-capture/components/form-steps.jsx

```diff
@@ -240,7 +243,7 @@ function FormSteps({
           return fields.current[field].refCallback;
         }}
       />
-      <Button type="submit" isPrimary className="margin-y-5">
+      <Button type="submit" isPrimary className="margin-y-5" isVisuallyDisabled={!canContinue}>
```

Comment:
For `FormSteps` spec, we could possibly assert on the disabled appearance around here:

https://github.com/18F/identity-idp/blob/6809bd750cf4f566be30c2afc73fa56208183d7f/spec/javascripts/packages/document-capture/components/form-steps-spec.jsx#L243

Or similar to how we check for the logic between showing "Submit" and "Continue" here:

https://github.com/18F/identity-idp/blob/6809bd750cf4f566be30c2afc73fa56208183d7f/spec/javascripts/packages/document-capture/components/form-steps-spec.jsx#L94-L111

Reply:
Will add, thanks!

Reply:
I had some trouble getting those specs to work, even after adding a `validator` to the `STEPS`. The btn-disabled behavior is covered in the `document-capture-spec.jsx` so I think it'll be fine for now